lib/Sema/TypeCheckGeneric.cpp

```diff
@@ -33,19 +33,14 @@ using namespace swift;
 
 /// Check the generic parameters in the given generic parameter list (and its
 /// parent generic parameter lists) according to the given resolver.
-void checkGenericParamList(TypeChecker &tc,
-                           GenericSignatureBuilder *builder,
-                           GenericParamList *genericParams,
-                           GenericSignature *parentSig,
-                           TypeResolution resolution) {
+static void checkGenericParamList(TypeChecker &tc,
+                                  GenericSignatureBuilder *builder,
+                                  GenericParamList *genericParams,
+                                  GenericSignature *parentSig,
+                                  TypeResolution resolution) {
   // If there is a parent context, add the generic parameters and requirements
```

Comment:
Maybe it would be appropriate to assert that builder is not a `nullptr` here just to be on the safe side?

Reply:
I don’t think that’s necessary. We will hit a segfault immediately if it’s null.